canvas_modules/common-canvas/src/common-properties/controls/expression/expression.scss

```diff
@@ -65,12 +66,28 @@
 			}
 		}
 		.validateLink {
-			min-height: 34px;
+			min-height: 32px;
 			padding: 4px;
 		}
 	}
 }
 
+.properties-expression-header {
+	position: relative;
+	height: 34px;
+	.properties-expression-title {
+		@include carbon--type-style("label-01");
+		position: absolute;
+		bottom: 5px;
```

Comment:
Why 5px?  This seems odd.

Reply:
The text needed to be in-line with the "Validate" button on the right side, which had 4px padding and 1px border on the bottom
![image](https://user-images.githubusercontent.com/10677021/93258621-737adf00-f753-11ea-89c3-404b8d525cef.png)
![image](https://user-images.githubusercontent.com/10677021/93258648-7e357400-f753-11ea-89c3-1890717b19f5.png)